blocks/library/embed/index.js

```diff
@@ -10,6 +10,11 @@ import './style.scss';
 import { registerBlockType, query } from '../../api';
 import Editable from '../../editable';
 
+/**
+ * External dependencies
+ */
+import { parse } from 'url';
```

Comment:
Minor: There's a bit of an undocumented ([semi-documented](https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#imports)?) convention around ordering these blocks `External` -> `WordPress` -> `Internal` (from most external to least external).